src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -196,13 +199,12 @@ def __enter__(self):
         self._inputs_queue = Queue()
         # Starting a new process in non-fork mode requires to allocate memory. Determine the maximum number of processes
         # based on available memory to avoid memory bursting.
-        if not self._use_fork:
```

Comment:
These two lines should be moved into _determine_worker_count():
![image](https://github.com/microsoft/promptflow/assets/113209/029aa8f5-5c6b-4cd8-81af-667cc71b8bbe)

Reply:
fixed